apps/src/javalab/JavalabEditor.jsx

```diff
@@ -30,62 +31,27 @@ import FontAwesome from '@cdo/apps/templates/FontAwesome';
 import _ from 'lodash';
 import msg from '@cdo/locale';
 import javalabMsg from '@cdo/javalab/locale';
+import HeightResizer from '@cdo/apps/templates/instructions/HeightResizer';
 import {CompileStatus} from './constants';
 import {makeEnum} from '@cdo/apps/utils';
 
-const style = {
```

Comment:
Diff for anyone interested
![image](https://user-images.githubusercontent.com/8324574/121094855-1cff7600-c7a4-11eb-9140-96f2c503559f.png)

Reply:
For more context on the name- Maureen's style update looked for all instances of "styles", so I figured I would rename this here for consistency.